instrumentation/net/http/otelhttp/common.go

```diff
@@ -31,10 +31,17 @@ const (
 
 // Server HTTP metrics.
 const (
-	RequestCount          = "http.server.request_count"           // Incoming request count total
-	RequestContentLength  = "http.server.request_content_length"  // Incoming request bytes total
-	ResponseContentLength = "http.server.response_content_length" // Incoming response bytes total
-	ServerLatency         = "http.server.duration"                // Incoming end to end duration, milliseconds
+	ServerRequestCount          = "http.server.request_count"           // Incoming request count total
+	ServerRequestContentLength  = "http.server.request_content_length"  // Incoming request bytes total
+	ServerResponseContentLength = "http.server.response_content_length" // Incoming response bytes total
+	ServerLatency               = "http.server.duration"                // Incoming end to end duration, milliseconds
+)
+
+// Client HTTP metrics.
+const (
+	ClientRequestContentLength  = "http.client.request_content_length"  // Outgoing request bytes total
+	ClientResponseContentLength = "http.client.response_content_length" // Outgoing response bytes total
```

Comment:
These do not match https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/metrics/semantic_conventions/http-metrics.md#http-client

Can we also fix the metrics for HTTP server instrumentation?

Reply:
0c097201a95fe872a4b19ba097f095cdae4353a7